artistools/inputmodel/maptogrid.py

```diff
@@ -429,12 +440,32 @@ def addargs(parser: argparse.ArgumentParser) -> None:
     parser.add_argument(
         "-ncoordgrid", type=int, default=50, help="Number of grid positions per axis (numcells = ncoordgrid^3)"
     )
+    parser.add_argument(
+        "-dtextra_seconds",
+        type=float,
+        default=0.5,
+        help="Time in seconds to propagate SPH particles ballistically after end of SPH simulation."
+        " 0 for no extrapolation",
+    )
+    parser.add_argument(
+        "-setgrid_fractionrmax",
```

Comment:
I suggest something like -rmax_over_maxparticleradius, but maybe it would be better for us specify a velocity cut in units of c (keeping this mode for compatibility with published models?)

Reply:
I find rmax_over_maxparticleradius much harder to understand what that does.
Yes, specifying a velocity would be better